templates/kapeta/block-type-frontend/webpack.development.config.js.hbs

```diff
@@ -105,7 +105,7 @@ const config = {
             "react": Path.resolve(__dirname, "./node_modules/react"),
             "react-dom": Path.resolve(__dirname, "./node_modules/react-dom")
             {{#consumes 'kapeta/resource-type-rest-client'}},
-            './browser.js': Path.resolve(__dirname, './src/mocks/browser.ts'),
+            './browser.js': Path.resolve(__dirname, './src/mocks/.generated/browser.ts'),
```

Comment:
I think you need to run `npm run test` again to get this change in the generated test code as well